ui/icon-set/svg-material-icons.js

```diff
@@ -6,10 +6,33 @@
  */
 
 
+import {
+  outlinedRning,
+  outlinedFo,
+  outlinedAdient,
+  outlinedNe,
+  outlinedYle,
+  outlinedFresh,
+  outlinedNs,
+  outlinedNcel,
+  outlinedEck,
+  outlinedDay,
+  outlinedNk,
+  outlinedLlscreen,
+  outlinedInt,
+  outlinedMove,
+  outlinedDo,
+  outlinedDe,
+  outlinedD,
+  outlinedOse,
+  outlinedRor,
+  outlinedAde,
+  outlinedIt,
+  outlinedEar
+} from '@quasar/extras/material-icons-outlined'
+
 import {
   matCheckCircle,
-  matWarning,
```

Comment:
Material versions should probably be used instead

Reply:
Yes, there are already dedicated icon sets for other styles. If there are enough icons in the classic material version, all of the icons must be selected from them to avoid a mixture.

Reply:
The fact is, there are 1293 icons now. More than they even have posted on their web site. And, their font versions still have some issues. Look at the `add_call` in this pen: https://codepen.io/Hawkeye64/pen/BaBQoQp